squarelet/users/viewsets.py

```diff
@@ -92,6 +101,33 @@ def retrieve(self, request, pk=None):
         return Response(sesame.utils.get_parameters(user))
 
 
+class PressPassRegisterView(RegisterView):
+    def get_response_data(self, user):
+        if (
+            allauth_settings.EMAIL_VERIFICATION
+            == allauth_settings.EmailVerificationMethod.MANDATORY
+        ):
+            return {"detail": _("Verification e-mail sent.")}
+        return {}
+
+    def perform_create(self, serializer):
+        data = serializer.get_cleaned_data()
+        data["name"] = ""
```

Comment:
Why no name?

Reply:
It's not included in `django-rest-auth`'s serializer, so we need to explicitly set it on the dict;